developer/ruby/FindAndAddMissingIPUnits.rb

```diff
@@ -0,0 +1,99 @@
+require '/home/julien/Software/Others/OS-build/Products/ruby/openstudio'
+require 'optparse'
+
+# define the input parameters
+options = Hash.new
+
+optparse = OptionParser.new do |opts|
+
+  opts.on("-f", "--[no-]fix", "Add the missing ipUnits to the OpenSutdio.idd (note: LOTS of whitespace changes...)") do |fix|
```

Comment:
This would require running the "iddWrapper.save" once to do the whitespace changes once. I don't think it's really worth it here...